lib/kafka_ex/network/client.ex

```diff
@@ -0,0 +1,67 @@
+defmodule KafkaEx.Network.Client do
+  require Logger
+
+  alias KafkaEx.Network.{TCP, Error}
```

Comment:
This is not supported in elixir < 1.2 and we have to support elixir > 1.0 please use the longer form